### DISCUSSION TITLE
ice can use js as server?

### DISCUSSION THREAD

**Discussion by starryfei** [zeroc-ice/ice-demos#135] [Q&A]
I want to implement callback functions， like[ java callback](https://github.com/zeroc-ice/ice-demos/tree/3.7/java/Ice/callback) demo. But I didn't find any examples of using JS as a server， 

Thank you very much for your answer

**Comment by pepone** [comment]
Hi, 

JavaScript mapping server side support is limited to bidirectional connections, you can receive callback invocations but you cannot listen for incoming connections.

See https://doc.zeroc.com/ice/3.7/language-mappings/javascript-mapping for a list of the features supported with the JavaScript mapping
